### ISSUE TITLE
pushing from termui gives weird ssh error.

### ISSUE THREAD

**Issue by DaveDavenport** [MichaelMure/git-bug#687]
I am on 663c429605b66d1947d0b7cc50a11cd142494bb2.
From termui trying to push a change I got the following error:

```
knownhosts: /home/user/.ssh/known_hosts:35: ssh:
exponent too large
```

From the commandline it worked fine (weird).
```
git bug bridge push
changed status: 7ed608976c50f2ed17230be222336af6cfbc84d2be59aa1ae7888e30511f274b
exported 0 issues with default bridge
```

Line 35 is valid in my known_hosts.

Not sure if I can help reproduce this, but thought it was an odd error.

**Comment by GlancingMind** [comment]
I think you are using two different operations here. I'll try to explain.
1. `git bug bridge push/pull`: Export and import bugs to some external bug tracker e.g. GitHubs Issue Tracker or Jira.
2. `git-bug push`: This commands works without a configured bridge. It  pushes the "raw" bug database (stored in your local .git directory) to the remote *origin* repository. In effect `git-bug push` is a disguised `git push` :-).

As far as I know, the push/pull operations in the TermUI only support `git-bug push/pull` commands. So the TermUI doesn't make use of your configured bridges.
TLDR: exporting and importing of bugs from another bug tracker seem to be not supported by the TermUI. So if you want to manage your GitHub Issues via git-bug, you have to use the commandline with `git-bug bridge push/pull`.

---

To your error. The second command `git bug bridge push` succeeds as ssh is not used by the bridge.
While your first command `TermUI::Push` uses ssh to `git push` your git-bug database to your remote repository.

Unfortunately this might not solve your error with the TermUI, but does calling `git-bug push` fail also with the same error as the TermUI push?

**Comment by DaveDavenport** [comment]
aah this was not directly clear to me from the ui. It does make sense.
I expected (a bridge) to be transparent.

**Comment by MichaelMure** [comment]
Closing as it's an quite old issue. Please reopen if that's still a problem.
